QualificationsFramework/Examples/ACQF.json

```diff
@@ -0,0 +1,184 @@
+			"ceterms:alternateName": { "en": "ACQF" },
+			"ceasn:description": { "en": "The ACQF is a comprehensive and inclusive meta-referencing qualifications framework, designed to support a holistic and systemic vision of learning, qualifications, and credentials. The ACQF is oriented to lifelong learning and supports parity of esteem of learning outcomes acquired in formal, non-formal and informal contexts, and all levels of education and training." },
+			"ceasn:dateValidFrom": "2022-11-01",
+			"ceterms:versionCode": "17",
+			"ceasn:source": [
+				"https://drive.google.com/file/d/1BnU5VOqS6hu2YRhowC7x6J1R1_p6UblV/view"
+			],
+			"ceasn:hasGlossary": [
+				"res:acqfGlossary"
+			],
+			"asn:hasProgressionModel": [
+				"res:acqfLevels"
+			]
+		},
+		{
+			"@id": "res:acqfGlossary",
+			"@type": "skos:ConceptScheme",
+			"ceasn:name": { "en": "ACQF Acronym Glossary" },
+			"skos:hasTopConcept": [
+				"res:acqfGlossary_acqf",
+				"res:acqfGlossary_acts",
+				"etc"
+			]
+		},
+		{
+			"@id": "res:acqfGlossary_acqf",
+			"@type": "skos:Concept",
+			"skos:notation": "ACQF",
+			"skos:prefLabel": { "en": "African Continental Qualifications Framework" },
+			"skos:inScheme": [
+				"res:acqfGlossary"
+			]
+		},
+		{
+			"@id": "res:acqfGlossary_acts",
+			"@type": "skos:Concept",
+			"skos:notation": "ACTS",
+			"skos:prefLabel": { "en": "African Credit and Transfer System" },
+			"skos:inScheme": [
+				"res:acqfGlossary"
+			]
+		},
+		{
+			"@id": "res:acqfFacets",
+			"@type": "skos:ConceptScheme",
+			"ceasn:name": { "en": "Domains of Learning" },
+			"ceasn:description": { "en": "Definition of the domains of learning" },
+			"ceasn:hasTopConcept": [
+				"res:facetKnowledge",
+				"res:facetSkills",
+				"res:facetAutonomy"
+			]
+		},
+		{
+			"@id": "res:facetKnowledge",
+			"@type": "skos:Concept",
+			"skos:prefLabel": { "en": "Knowledge" },
+			"skos:definition": { "en": "Includes various kinds of knowledge such as facts, principles, and theories in various areas. The learning outcomes at all ten ACQF levels relate to formal, non-formal and informal learning. The knowledge domain comprises two sub-domains or elements:\n- Type of knowledge\n- Scope of knowledge" },
+			"skos:note": { "en": "In the context of ACQF \"Knowledge\" includes various kinds of knowledge such as facts, principles, and theories in various areas" }
+		},
+		{
+			"@id": "res:facetSkills",
+			"@type": "skos:Concept",
+			"skos:prefLabel": { "en": "Skills" },
+			"skos:definition": { "en": "Refer to the ability to use knowledge to respond to information and address problems/ issues. Skills include cognitive, communication, digital, green, innovation, practical and social skills. The learning outcomes at all ten ACQF levels relate to formal, non-formal and informal learning. The skills domain comprises three sub-domains:\n- Type of skills required\n- How the skills are used to respond to information\n- How skills are used to address types of problems" },
```

Comment:
Remove extra space after `problems/`?

Reply:
I'm not opposed to it, but the space is there in the PDF (mid-line, so it's not an artifact of a line break).  We should discuss the impact of fixing (or not) these kinds of minor typographical things with @philbarker since part of what we want to do is capture the data as-is.